src/BitcoinHelpers.js

```diff
@@ -711,8 +711,19 @@ const BitcoinHelpers = {
           const outpointMatches = (
             await electrumClient.getTransaction(tx.tx_hash)
           ).vin.some(({ txid, vout }) => {
-            const inputOutpoint = txid + vout
```

Comment:
Man in retrospect this was a real amateur move where I got absolutely eaten by the fact that concatenating a number to a string is valid in JS. Wowie.

Separately, m I a monster for thinking that this is easier to implement by using the buffer solely for big-endianing the vout? In particular, continuing to use string concatenation for the txid + vout combo and continuing to use string comparison for the outpoint equality? Or is that not going to work for other reasons?

Reply:
Additionaly to vout transforming we are also using buffer to reverse bytes in the combined outpoint. I'm not sure which way shall we take to handle these transformations in the most elegant way.

Reply:
Monstrous. Why do people do this to themselves.